seqr/fixtures/1kg_project.json

```diff
@@ -2059,7 +2059,7 @@
             "liftedOverChrom": null,
             "svType": "DEL",
             "variantId": "prefix_19107_DEL",
-            "chrom": "12",
+            "chrom": "1",
```

Comment:
this saved variant fixture actually had a discrepancy between what the xpos represented for its position and what this chrom/pos in the saved_variant_json represented, so this fixes it so they are the same